text/getting_started.tex

```diff
@@ -129,23 +132,24 @@ \section{Setting up a development environment}
 \ref{vscode} shows the result in Visual Studio Code.
 All of the CHERIoT-specific extensions (see \ref{language_extensions}) are correctly highlighted.
 
-This support is not limited to Visual Studio Code, it can work with any editor that supports the language-server protocol.
+This support is not limited to Visual Studio Code. It can work with any editor that supports the language-server protocol.
 The parsing code from the clang front end is also part of the \command{clangd} dæmon, which implements the server part of this protocol.
 
 The dev container includes a \file{.vimrc} that (if you install vim) uses the \href[src=https://github.com/dense-analysis/ale]{Asynchronous Lint Engine} plugin to connect to our \command{clangd} build.
 Simply run \command{:PlugInstall} in Vim to install it.
+\amanda{I was unable to find a definitive answer to whether vim should be capitalized or not, but you should be consistent and either capitalize the first instance of it ("...if you install vim") or uncapitalize the second instance (..."in Vim to install it.")}
```

Comment:
Vim should be capitalised (proper noun for the program), `\command{vim}` should not (the thing that you type on the command line to run it).